javascript/javascript1/week2/week2-guzide/student-manager/StudentManager.js

```diff
@@ -0,0 +1,37 @@
+const class07Students = [];
+
+function getNumberOfStudents() {
+  return class07Students.length;
+}
+
+function addStudentToClass(studentName) {
+  if (studentName == "") {
+    console.log("You can't leave this area empty");
```

Comment:
It is good to write explicit and clear error messages - I think something like "Student name cannot be empty" would work better. It is bit hard to say what "this area" refers to specifically.

Reply:
Yes my bad, thank you.